search/tasks.py

```diff
@@ -45,7 +45,7 @@ def wrap_retry_exception(*exception_classes):
     except Exception as ex:
         if isinstance(ex, NotFound):
             # No corresponding reddit post/comment found for django model object.  Log error and continue indexing.
-            log.exception(str(ex))
+            log.exception()
```

Comment:
Actually it looks like this requires an argument. Can you add a brief message? The exception stacktrace will be logged before it automatically

Reply:
✔️